site/composer.json

```diff
@@ -34,7 +34,7 @@
 		"exercise/htmlpurifier-bundle" : "*",
 		"doctrine/doctrine-migrations-bundle" : "^1.0",
 		"symfony/http-kernel" : "~3.2.3",
-		"knplabs/github-api": "^2.3",
+		"knplabs/github-api": "dev-master",
```

Comment:
Yes, but it stores a commit sha in composer.lock, so everyone will get the same version.
We need some features (GraphQL GA) and it has not been released yet. It's mostly a temporary solution, we will pin a version number in composer.json, once the v2.6 is released.